### ISSUE TITLE
Crash when not inputting a number when selecting song 

### ISSUE THREAD

**Issue by jamesblckwell** [deepjyoti30/ytmdl#123]
# Bug report

## Checklist

[ Y] I have searched for similar bugs in the issues.
[ Y] I will add the bug label to this issue.

## Version Info

- Ytmdl version: `2020.11.20-1`
- Youtube-dl version: `2020.11.19`

## How Ytmdl was installed?

Installed via AUR/Yay

## Console Log

```console
   Music ytmdl wave deca joins --level DEBUG
 ==>  Logger running in DEBUG mode 
 ==>  Passed args: Namespace(SONG_NAME=['wave', 'deca', 'joins'], album=None, artist=None, choice=None, disable_file=False, disable_metaadd=False, format='mp3', get_opts=False, ignore_errors=False, level='DEBUG', list=None, list_level=False, manual_meta=False, nolocal=False, pl_end=None, pl_items=None, pl_start=None, proxy=None, quiet=False, skip_meta=False, song=None, trim=False, url=None) 
 ==>  ['wave', 'deca', 'joins'] 
 ==>  wave deca joins 
 ==>  Searching to see if already present in /home/jamesb/Music 
 ==>  proxy passed: None 
 ==>  Passed format: mp3 
 ==>  Searching Youtube for wave deca joins 
 ==> Choose One (One with [M] is verified music)
 [1] deca joins | 海浪【Official Music Video】 by deca joins with dur 5:32
 [2] Deca Joins - 海浪 Wave | 2019 秋OUT [LIVE 4K] by Joanne조앤 with dur 5:19
 [3] deca joins - 海浪｜新倫Xinlun《吉他cover + tabs》p.s 影片後有心得分享 by 新倫 XinLun with dur 10:35
 [4] [Full Album] deca joins  - Go Slow by TE JUNG LEE with dur 20:07
 [5] deca joins | 浴室【Official Music Video】 by deca joins with dur 4:40
 [0] More results
 ==> Enter Choice [default is 1] -
Traceback (most recent call last):
  File "/usr/bin/ytmdl", line 469, in <module>
    extract_data()
  File "/usr/bin/ytmdl", line 464, in extract_data
    main(args)
  File "/usr/bin/ytmdl", line 316, in main
    choice = song.getChoice(data, 'mp3')
  File "/usr/lib/python3.8/site-packages/ytmdl/song.py", line 148, in getChoice
    choice = int(choice)
ValueError: invalid literal for int() with base 10: '-'
```

## Description

YTMDL crashes when you input anything other than an integer during selection of song. I assume it should just re-try and ask you to input a number instead.

**Comment by deepjyoti30** [comment]
@jamesblckwell Thanks for the input. This bug was supposed to be fixed in one of the earlier commit messages but it will surely be fixed before the next release. Since this is not a major issue, I can keep it for the next release.

**Comment by deepjyoti30** [comment]
Fixed in 05a4d9cd41a85813b8

@jamesblckwell You can try using the above commit and report back if the issue still exists. For now I will close the issue. Feel free to open it if the issue remains.
